style.css

```diff
@@ -196,27 +204,47 @@ input{
 	border: none; 
 	border-bottom: 0.25vw solid #527ECC;
 	padding: 0.4vw 0.6vw;
-	width: 12vw;
+	width: 20.5vw;
 	font-family: 'Montserrat', sans-serif;
 	font-size: 1.2vw;
+	font-weight: 600;
+	color: #1155CC; 
 	outline-width: 0;
 }
 
 input[type=password] {
   	border: none; 
 	border-bottom: 0.25vw solid #527ECC;
 	outline-width: 0;
-	width: 12vw;
+	width: 20.5vw;
 	font-family: 'Montserrat', sans-serif;
 	font-size: 1.2vw;
+	font-weight: 600;
+	color: #1155CC; 
+}
+
+/* changes the input styling when clicked by the user. */
+input:focus {
```

Comment:
#added technique that changes the colour of the bottom border when input box is clicked on